softvis3d-frontend/checkTestFileLocationAndName.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
* Do we still need this script?
* Should long-term scripts be written in shell/bash or in JavaScript/TypeScript and executed with NodeJs?